eal/cluster_configuration.ex

```diff
@@ -33,6 +33,15 @@ defmodule AntikytheraEal.ClusterConfiguration do
     This callback is called only at startup and the return value is passed to `RaftFleet.activate/1`.
     """
     @callback zone_of_this_host() :: String.t()
+
+    @doc """
+    Returns the number of seconds for which the initial health check to this host is delayed to
+    provide ample warm-up time for the host.
+
+    This callback is called only at startup and the return value is used to calculate the trial count for
+    establishing connections to other nodes.
+    """
+    @callback health_check_grace_period() :: non_neg_integer
```

Comment:
Could you rename this to `health_check_grace_period_in_seconds` ?

Reply:
Sure!
About a9a code, fixed in https://github.com/access-company/antikythera/pull/168/commits/80535060259709834093d8cb91df9842d2b21f31
About a9a_aws code, fixed in https://github.com/access-company/antikythera_aws/pull/8/commits/ec8a3a14799e6b4428bb0544dbeba7f46388e687